.github/workflows/ci-core-lint-reusable.yml

```diff
@@ -36,5 +36,5 @@ jobs:
           ci_run zk lint js --check
           ci_run zk lint ts --check
           ci_run zk lint md --check
-          ci_run zk db check-sqlx-data
 
+# ci_run zk db check-sqlx-data
```

Comment:
Debug artifact?

Reply:
No, this check will not work because our CI images are still those generated with 0.5.* and they are not compatible. So we have to merge this -> images get built -> we bring this back

Reply:
Add indentation please